### PR TITLE
redesigned template for the panel

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts [hunk 1]
```diff
@@ -167,7 +178,18 @@ export class DatasetViewMenuComponent implements OnInit, AfterViewInit {
     }
 
     private initClipboardHints(): void {
-        this.clipboardKamuCli = `kamu pull kamu.dev/${this.datasetBasics.alias}`;
-        this.clipboardKafka = `https://api.kamu.dev/kafka/${this.datasetBasics.alias}`;
+        this.clipboardReference = `https://kamu.dev/${this.datasetBasics.alias}`;
```
Comment (sergiimk) [inline src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts:181]: We should un-hardcode `kamu.dev` - frontend host should come from browser URL
(as a temporary solution until correct URLs are provided via API)

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts [hunk 2]
```diff
@@ -167,7 +178,18 @@ export class DatasetViewMenuComponent implements OnInit, AfterViewInit {
...
+        this.clipboardKamuCliPull = `kamu pull kamu.dev/${this.datasetBasics.alias}`;
+        this.clipboardKamuCliPush = `kamu push kamu.dev/${this.datasetBasics.alias}`;
+        this.clipboardKafka = `- coming soon -`;
+        this.clipboardRestUrlTail = `https://node.demo.kamu.dev/kamu/${this.datasetBasics.alias}/tail?limit=10`;
```
Comment (sergiimk) [inline src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts:185]: We should un-hardcode `node.demo.kamu.dev` - backend host should come from api-server configuration we already use for GQL
(as a temporary solution until correct URLs are provided via API)

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html [hunk 3]
```diff
@@ -217,13 +729,66 @@
...
-                                    <button class="select-menu-item">DOI</button>
-                                    <button class="select-menu-item">Bibtech</button>
+                                <mat-tab>
+                                    <ng-template mat-tab-label>
+                                        <div class="d-flex align-items-center">
+                                            <mat-icon class="text-muted fs-5 mt-2">ios_share</mat-icon>
+                                            Export
+                                        </div>
+                                    </ng-template>
+                                    <button mat-menu-item class="select-menu-item" [disabled]="true">as Parquet</button>
```
Comment (sergiimk) [inline src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html:787]: Let's replace format selection with a drop-down and add a "Dowload" button (disabled for now)

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html [hunk 4]
```diff
@@ -90,12 +90,12 @@
         <li class="nav-item mr-2">
             <div class="btn-group-parent">
                 <button
-                    class="dataset-download-button starred rounded-left-2 border-1 border-right-0 btn-sm btn btn-group-item d-flex justify-content-center align-items-center"
+                    class="border-1 border btn-sm btn btn-group-item d-flex justify-content-center align-items-center"
```
Comment (sergiimk) [inline src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html:93]: Please rename the "Download" button to "Get Data", and let's make it green - more visible.

Also, let's make the whole button clickable, not just its side arrow.

image

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html [hunk 5]
```diff
@@ -106,60 +106,291 @@
...
-                                            (click)="$event.stopPropagation()"
-                                            (keydown)="$event.stopPropagation()"
-                                        >
+                                        <div class="input-group mb-1">
+                                            <span class="fs-12 text-muted"> Share this dataset on the Web:</span>
+                                        </div>
+
+                                        <div class="select-menu-item align-items-start p-0 border-0 mb-2">
+                                            <div class="input-group">
+                                                <input
```
Comment (sergiimk) [inline src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html:142]: I can currently edit these URLs - they should be made read-only.

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts (hunk 1 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts:181]: Changed  to

> this.clipboardReference = `https://${location.host}/${this.datasetBasics.alias}`;

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts:185]: Changed to  

> this.clipboardRestUrlTail = `${this.appConfigService.apiServerGqlUrl}/${this.datasetBasics.alias}/tail?limit=10`;

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html (hunk 4 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html:93]: Fixed

image

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html (hunk 5 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html:142]: Fixed

File: src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html (hunk 3 above)
Comment (dmitriy-borzenko) [reply src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html:787]: Fixed 

image

**Review by zaychenko-sergei — COMMENTED** [review]
Plz extract the panel as as separate component. Later we will re-structure it's internals and attach an API, but at least we would not pollute into a current clean component.

**Pull request merged**